### DISCUSSION TITLE
[Feature] Import from Tripit

### DISCUSSION THREAD

**Discussion by flosoft** [johanohly/AirTrail#17] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be great to be able to import data from Tripit.

**Comment by johanohly** [comment]
I have seen multiple people mentioning TripIt, but it will also require a bit more work than for other sources, as TripIt doesn't provide a full data export option. 
For now I will let this feature garner upvotes to see how wanted this feature is.

**Reply by T-h-o-m-a-s** [reply]
Natively from Tripit would be amazing. 
But via https://www.tripexporter.com could also work as a start.

**Comment by Deekerman** [comment]
This would be amazing!

**Comment by flosoft** [comment]
It seems like Tripit has some API documentation here - http://tripit.github.io/api/

**Comment by johanohly** [comment]
For now it is possible to import individual trip exports from TripIt into AirTrail. Hope this helps.
